perfkitbenchmarker/linux_benchmarks/dpb_df_template_benchmark.py

```diff
@@ -0,0 +1,200 @@
+# Copyright 2022 PerfKitBenchmarker Authors. All rights reserved.
```

Comment:
Kind of pedantic comment: I'd rather name this file `dpb_dataflow_streaming_template_benchmark.py`.

It's a long name, but here are my reasons:
- `df` is confusing, if you don't have context on Dataflow.
- This uses a streaming template, we might add support for non batch templates in the future.

Pleas update all referrences accordingly.

Reply:
Makes sense. will update benchmark name.